src/Logic/LanguageDeserializer.cs

```diff
@@ -19,7 +19,8 @@ public static Language CustomDeserializeLanguage(string fileName)
             var stream = new FileStream(fileName, FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
             var language = new Language();
 
-            using (XmlReader reader = XmlReader.Create(stream))
+            using (XmlReader reader = XmlReader.Create(stream, new XmlReaderSettings() {
+                   IgnoreWhitespace = true, IgnoreProcessingInstructions = true, IgnoreComments = true, CloseInput = true }))
```

Comment:
Cool, this seems slightly faster: 
'using (XmlReader reader = XmlReader.Create(stream, new XmlReaderSettings { IgnoreWhitespace = true, IgnoreProcessingInstructions = true, IgnoreComments = true, DtdProcessing = DtdProcessing.Ignore, CheckCharacters = false }))'